### DISCUSSION TITLE
How to add basic auth to delta fuseki?

### DISCUSSION THREAD

**Discussion by ll13j1209** [afs/rdf-delta#198] [Q&A] [answered]
Hi, how can we config delta fuseki so that it supports basic authentication with username and password for security? Thank you

**Comment by afs** [comment, marked as answer]
Delta Fuseki is the normal Apache Jena Fuseki Main with datasets that understand patch handling.

https://jena.apache.org/documentation/fuseki2/fuseki-data-access-control.html

`--auth basic --passwd=_<jetty passwd file>_`

`--help` will show you the full list of arguments.

**Comment by ll13j1209** [comment]
Great! Thank you for the explanation!
